app/src/main/java/com/fcaputo/parkingapp/mvp/contract/HomeContract.kt

```diff
@@ -4,12 +4,15 @@ interface HomeContract {
     interface Presenter {
         fun onSettingsClick()
         fun onMakeReservationClick()
+        fun onReleaseClick()
```

Comment:
As a suggestion, the standard is to use the word `'Pressed'` instead of `'Click'`.